docs/introduction/hello-world.md

```diff
@@ -86,6 +86,10 @@ $ ./hello_world_receiver
 You should see `Hello, World` printed out in the terminal where you ran
 the `hello_world_receiver`.
 
+*For a guide to run this example in Visual Studio go to this link:
+[hello-world-VS](https://helics--1621.org.readthedocs.build/en/1621/introduction/hello-world-VS.html).*
```

Comment:
This should probably be a relative link vs a link the PR generated doc, I am not sure how to generate that though right now

Reply:
Lets fix this link to something that is generated then I think we can merge this

Reply:
Relative links to markdown files should get replaced with the correct generated link when RTD generated the html files, e.g. `[hello-world-VS](./hello-world-VS.md)`